docs/architecture/cloud-native/index.md

```diff
@@ -39,15 +39,17 @@ All other marks and logos are property of their respective owners.
 
 Authors:
 
-> **Steve "ardalis" Smith** - Software Architect and Trainer - [Ardalis.com](https://ardalis.com)
->
 > **Rob Vettor** - Microsoft - Principal Cloud System Architect/IP Architect - [thinkingincloudnative.com](http://thinkingincloudnative.com/about/)
```

Comment:
To be consistent with the other entries. Is IP here an acronym for Intellectual Property?

```suggestion
> **Rob Vettor**, Principal Cloud System Architect/IP Architect - [thinkingincloudnative.com](http://thinkingincloudnative.com/about/), Microsoft
```

Reply:
Done